modules/Hoot/ui/conflicts/merge.js

```diff
@@ -342,4 +340,34 @@ export default class Merge {
 
         Hoot.context.background().updateArrowLayer( gj );
     }
+
+    checkMergeButton() {
+        let features = _clone( this.data.currentFeatures ),
+            toFeature = Hoot.context.hasEntity( features[0].id ),
+            fromFeature = Hoot.context.hasEntity( features[1].id ),
```

Comment:
Use dynamic features retrieved from graph.  This allows the show/hide merge button to work on undo/redo.